src/nl/rubensten/texifyidea/TexifyIcons.java

```diff
@@ -178,6 +178,13 @@
             "/nl/rubensten/texifyidea/icons/dot-env.png"
     );
 
+    /**
+     * Copyright (c) 2018 Ruben Schellekens
+     */
+    public static final Icon DOT_CLASS = IconLoader.getIcon(
+            "/nl/rubensten/texifyidea/icons/dot-cls.png"
```

Comment:
Did you push this file? I can't find it anywhere, and especially not in the place it should be. It may be the reason why the plugin is broken for me.